### PR TITLE
Small fix that prevents accessing uninitialised memory.

### PR DESCRIPTION
**Pull request by pranomostro**
When a friend request with an empty message is created, req->msg would not be initialised.
This is fixed here.

### CODE REVIEW

**Comment by GrayHatter** [comment]
you missed a `;`

File: ratox.c [hunk 1]
```diff
@@ -639,6 +639,8 @@ cbfriendrequest(Tox *UNUSED(m), const uint8_t *id, const uint8_t *data, size_t l
 			eprintf("malloc:");
 		memcpy(req->msg, data, len);
 		req->msg[len] = '\0';
+	} else {
+		req->msg="ratox is awesome!";
```
Comment (iphydf) [inline ratox.c]: Add spaces around `=`.

Then, rebase on current master.

**Comment by iphydf** [comment]
Thanks, please squash.

**Comment by pranomostro** [comment]
Could you merge? Because I have some own development going on in my ratox repo, and squashing there introduces some conflicts that are cumbersome to resolve.

**Comment by pranomostro** [comment]
Thanks.

**Pull request merged**